Makefile

```
@@ -432,6 +432,16 @@ nix-install: ## Install Nix if not already installed.
 	fi
 	@echo "✅ Nix environment installed!"
 
+.PHONY: nix-cache-warmup
+nix-cache-warmup: nix-connect nix-trust ## Warm flake input metadata before build/switch.
```

checkmake (0.3.2)

[warning] Target body for "nix-cache-warmup" exceeds allowed length of 5 lines (7).

(maxbodylength)